src/client.js

```diff
@@ -63,40 +62,20 @@ export async function getOneDriveClient() {
     info(chalk`AZURE_APP_CLIENT_SECRET={gray <client-secret of the app>}`);
     process.exit(-1);
   }
-  let tokens = [];
-  try {
-    tokens = await fs.readJson(AUTH_FILE, 'utf-8');
-  } catch (e) {
-    // ignore
-  }
-
-  let {
-    AZURE_APP_REFRESH_TOKEN: refreshToken = '',
-  } = process.env;
-
-  if (!refreshToken && tokens.length) {
-    refreshToken = tokens[0].refreshToken;
-  }
-
```

Comment:
this is handled by the FSCachePlugin and the msal internal handling with refresh tokens